docs/concepts/services-networking/service.md

```diff
@@ -585,6 +585,84 @@ annotation:
 Since version 1.3.0 the use of this annotation applies to all ports proxied by the ELB
 and cannot be configured otherwise.
 
+#### ELB Access Logs on AWS
+
+There are several annotations to manage access logs for ELB services on AWS.
+
+The annotation `service.beta.kubernetes.io/aws-load-balancer-access-log-enabled`
+controls whether access logs are enabled.
+
+TODO: Describe each annotation here?
```

Comment:
Should I describe each annotation above an example?

Reply:
@micahhausler If this is still an open question, I think it'd be great. It's not a blocker, though.